src/itemset.js

```diff
@@ -180,7 +222,8 @@ define([
         },
         updateControlNodeAdaptorMap: function (map) {
             this.__callOld();
-            var adaptItemset = parser.makeControlOnlyMugAdaptor('Itemset');
+            var adaptItemset = parser.makeControlOnlyMugAdaptor('Itemset'),
+                use_advanced_itemsets = this.opts().features.advanced_itemsets;
```

Comment:
Annoyingly the places where I need to query this value are called before the init function: https://github.com/dimagi/Vellum/blob/master/src/base.js#L127-L149 so couldn't use a global